services/QuillLMS/spec/services/serialize_evidence_prompt_health_spec.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'SerializeEvidencePromptHealth' do
+  before do
```

Comment:
This particular before block setups a lot of files, but each spec uses the same setup.   I would collapse all of the specs into one spec and just match the whole results hash:

```
describe 'SerializeEvidencePromptHealth' do
  let(:expected_results) do
    {
      prompt_id: @because_prompt1.id,
      activity_short_name: @activity.notes,
      text: @because_prompt1.text,
      current_version: @activity.version,
      version_responses: 10,
      first_attempt_optimal: 25,
      final_attempt_optimal: 75,
      avg_attempts: 2.5,
      confidence: 0.85,
      percent_automl_consecutive_repeated: 0,
      percent_automl: 30,
      percent_plagiarism: 10,
      percent_opinion: 20,
      percent_grammar: 30,
      percent_spelling: 10,
      avg_time_spent_per_prompt: 287
    }
  end
  
  before do
    #keep same
  end

  it 'correct basic data for that prompt' do
    prompt = @because_prompt1
    data = SerializeEvidencePromptHealth.new(prompt, @prompt_feedback_history).data
    expect(data).to eq expected_results
  end
```

If you wanted to check for specific variations on values I could see adding additional specs.